clarity/src/vm/database/clarity_db.rs

```diff
@@ -1370,17 +1367,10 @@ impl<'a> ClarityDatabase<'a> {
         let value = Value::Tuple(
             TupleData::from_data(vec![
                 (
-                    ClarityName::try_from("reporter").map_err(|_| {
-                        InterpreterError::Expect("BUG: valid string representation".into())
-                    })?,
+                    ClarityName::from("reporter"),
```

Comment:
I made this change before and was told to revert it because `ClarityName::from()` can panic. See #4398

Reply:
Yup, this needs to be reverted.

Reply:
Reverted!